backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,71 @@
+[id="installing-oadp-kubevirt"]
+// If docs team approves the name, "Configuring OADP with Kubevirt" great but I suspect it will need to be "Configuring OADP with OpenShift Virtualization" or "Configuring OADP with Virtualization" or perhaps... bear with me.. "OADP and OpenShift-Virtualization"
+= Configuring the OpenShift API for Data Protection with {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-kubevirt
+:installing-oadp-kubevirt:
+:credentials: cloud-credentials
+:provider: kubevirt
+
+toc::[]
+
+You can install the {oadp-first} with {VirtProductName} by installing the OADP Operator and configuring a backup location. Then, you can install the Data Protection Application.
+
+// include::snippets/oadp-mtc-operator.adoc[]
+
+Back up and restore virtual machines by using the xref:../../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
+
+[NOTE]
+====
+(oadp-full} with {VirtProductName} backup and restore support is limited to:
+* Container Storage Interface (CSI) backups
+* Container Storage Interface (CSI) backups with DataMover
+
+This support excludes:
+* File System backup and restore.
+* Volume Snapshot Backups and Restores.
+
+For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Install the xref:../../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#about-installing-oadp[OADP Operator] according to the instructions for your storage provider.
+
+. Install the xref:oadp-installing-dpa-1-3_installing-oadp-kubevirt[Data Protection Application] with the `kubevirt` and `openshift` xref:../../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
+
+. Back up virtual machines by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
+
++
+[WARNING]
+====
+Red Hat support is limited to only the following options:
+* CSI backups
+* CSI backups with DataMover.
+====
+
+. Restore the `Backup` CR by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR].
+
+To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. For details, see xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
```

Comment:
```suggestion

```

Reply:
I see you're trying to do an Overview-type link situation here, but I think these kind of links need to be in == Additional resource heading like here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections